src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -136,6 +139,9 @@ const useCoreChatSystemHook = () => {
     websocketData: GetstreamWebsocket,
     channelCategory: MyChannelType
   ) => {
+    /**
+     * TODO: LEGACY FOLLOW SYSTEM USER, CHANGE TO NEW SYSTEM
+     */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This comment indicates that the following block of code is part of a legacy system and needs to be updated. It would be helpful if the developer provided more information about what the new system should look like or how this code should be refactored.
```diff
+    /**
+     * TODO: LEGACY FOLLOW SYSTEM USER, CHANGE TO NEW SYSTEM
+     */
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->